templates/answerfile_4.4_basic.txt.j2

```diff
@@ -1 +1,7 @@
 {% include "./templates/answerfile_4.3_basic.txt.j2" %}
+{% if ovirt_engine_apache_config_root_redirection is defined %}
+QUESTION/1/OVESETUP_APACHE_CONFIG_ROOT_REDIRECTION=str:{{ ovirt_engine_apache_config_root_redirection | ternary('yes','no')  }}
+{% endif %}
+{% if ovirt_engine_apache_config_ssl is defined %}
+QUESTION/1/OVESETUP_APACHE_CONFIG_SSL=str:{{ ovirt_engine_apache_config_ssl | ternary('automatic','manual') }}
+{% endif %}
```

Comment:
Can we add above changes to 4.3 template, so it's available for both 4.3 and 4.4?

Reply:
Moved the changes to answerfile_4.3 so it will be compatible with 4.3 and 4.4